src/main/java/org/jitsi/videobridge/octo/OctoPacket.java

```diff
@@ -21,22 +21,25 @@
 
 /**
  * A utility class which handles the on-the-wire Octo format. Octo encapsulates
- * its payload (RTP, RTCP, or anything else) in an 8-byte header:
+ * its payload (RTP, RTCP, or anything else) in an 12-byte header:
  * <pre>{@code
  *  0                   1                   2                   3
  *  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
  * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
- * |R| M | S | res |              Conference ID                    |
+ * |                         Conference ID                         |
  * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
  * |                         Endpoint ID                           |
  * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |MT|                      Reserved                              |
```

Comment:
nit: there should be a space before the closing `|` of the `MT` chunk to align it with the boundary of the 3rd bit